lib/Core/PTree.h

```diff
@@ -28,15 +33,28 @@ namespace klee {
   };
 
   class PTree {
+    // Number of registered ID
+    int registeredIds = -1;
```

Comment:
Meh... this semantics is ugly. How do I register a negative amount?

Reply:
What are you suggesting? It's just 0 semantics, that's usually cleaner.

To answer your questions: you can't, therefore there aren't any

Reply:
I'd expect a small unsigned type initialised with 0...

Reply:
But then the shift has an ugly `1 << (registeredIds + 1)`

Reply:
```
uint8_t id = 1 << registeredIds++;
...
return id;
```